go-runtime/ftl/ftltest/ftltest.go

```diff
@@ -102,8 +151,8 @@ func WithSecret[T ftl.SecretType](secret ftl.SecretValue[T], value T) func(*Opti
 //	... other options
 //
 // )
-func WhenVerb[Req any, Resp any](verb ftl.Verb[Req, Resp], fake func(ctx context.Context, req Req) (resp Resp, err error)) func(*Options) error {
-	return func(state *Options) error {
+func WhenVerb[Req any, Resp any](verb ftl.Verb[Req, Resp], fake func(ctx context.Context, req Req) (resp Resp, err error)) Option {
```

Comment:
I just thought of this, but we're going to need equivalent "When" options for `ftl.Sink`, `ftl.Source` and `ftl.Empty`.